src/test/java/step3/CarTest.java

```diff
@@ -3,20 +3,28 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.*;
 
 class CarTest {
 
     private Car car = new Car();
-    private ForwardTest forwardTest = new ForwardTest();
+    private CarMovement forwardTest = () -> true;
 
     @DisplayName("car 전진시 currnetPosition 증가 테스트")
     @Test
     void moveForwardTest(){
-        if(forwardTest.successForward()){
-            car.moveForward();
+        if(forwardTest.isSuccess()){
+            car.moveForward(new ForwardTest().isSuccess());
         }
```

Comment:
if(forwardTest.isSuccess()) 는 항상 true이므로 
이 코드는 아래와 같이 바꾸는게 어떨까요? 어차피 항상 true일때 전진하는걸 확인하고자 하는거니까요:)
```suggestion
        car.moveForward(forwardTest.isSuccess());
```